znvis/visualizer/headless_visualizer.py

```
@@ -0,0 +1,310 @@
+
+        This needs to be a seperate method so that the
+        image storing thread can run to completion before
+        this one is called. (GIL stuff)
+        """
+        images = [f.as_posix() for f in self.frame_folder.glob("*.png")]
+
+        # Sort images by number
+        images = sorted(images, key=lambda s: int(re.search(r"\d+", s).group()))
+
+        single_frame = cv2.imread(images[0])
+        height, width, layers = single_frame.shape
```

Ruff (0.14.0)

Unpacked variable `layers` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)